pygtk2/modules/support.py

```diff
@@ -1152,6 +1160,12 @@ def on_button_rotate_90_ccw_clicked(*args):
         img_parms.height = img_parms.width
         img_parms.width = new_width
         image_load_into_dialog()
+    def on_button_flip_horizontal_clicked(*args):
+        img_parms.original_pixbuf = img_parms.original_pixbuf.flip(true)
+        image_load_into_dialog()
+    def on_button_flip_vertical_clicked(*args):
+        img_parms.original_pixbuf = img_parms.original_pixbuf.false(false)
```

Comment:
thanks for your work first of all, this above is flip(false) I believe, not false(false) right?

Reply:
Yes. You are correct. It was a mistype.